code/game/gamemodes/endgame/scoreboard/scoreboard.dm

```diff
@@ -0,0 +1,317 @@
+	dat += science_score()
+	if(NO)
+		dat += nuke_op_score(NO)
+	if(RV)
+		dat += revolution_score(RV)
+	if(TR)
+		dat += syndicate_score(TR)
+	dat += silicon_score()
+	dat += misc_score()
+	
+	dat += display()
+	
+	round_end_info = dat
+	round_end_info_no_img = remove_images(dat)
+	log_game(round_end_info_no_img)
+	stat_collection.crew_score = score.crewscore
+	
+	to_chat(world, "<b>The crew's final score is:</b>")
+	to_chat(world, "<b><font size='4'>[score.crewscore]</font></b>")
+
+	for(var/mob/E in player_list)
+		E.display_round_end_scoreboard()
+	
+	ticker.mode.send2servers()
+	return
+
+/datum/controller/gameticker/scoreboard/proc/display(var/dat)
+	dat += {"<BR><h2>Round Statistics and Score</h2>"}
+	dat += {"<B><U>GENERAL STATS</U></B><BR>
+
+	<U>THE GOOD:</U><BR>
+	<B>Length of Shift:</B> [round(world.time/600)] Minutes ([round(score.time * 0.2)] Points)<BR>
+	<B>Shuttle Escapees:</B> [score.escapees] ([score.escapees * 100] Points)<BR>
+	<B>Random Events Endured:</B> [score.eventsendured] ([score.eventsendured * 200] Points)<BR>
+	<B>Meals Prepared:</B> [score.meals] ([score.meals * 5] Points)<BR>
+	<B>Hydroponics Harvests:</B> [score.stuffharvested] ([score.stuffharvested] Points)<BR>
+	<B>Ultra-Clean Station:</B> [score.messbonus ? "Yes" : "No"] ([score.messbonus] Points)<BR>
+	<B>Plasma Shipped:</B> [score.plasmashipped] ([score.plasmashipped * 0.5] Points)<BR>
+	<B>Centcom Orders Fulfilled:</B> [score.stuffshipped] ([score.stuffshipped * 100] Points)<BR>
+	<B>Ore Smelted:</B> [score.oremined] ([score.oremined] Points)<BR>
+	<B>Whole Station Powered:</B> [score.powerbonus ? "Yes" : "No"] ([score.powerbonus] Points)<BR>
+	<B>Isolated Vaccines:</B> [score.disease_vaccine] ([score.disease_vaccine_score] Points)<BR>
+	<B>Extracted Symptoms:</B> [score.disease_extracted] ([score.disease_effects] Points)<BR>
+	<B>Harvested Slimes:</B> [score.slimes] ([score.slimes * 20] Points)<BR><BR>
+	<B>Analyzed & Activated Large Artifacts:</B> [score.artifacts] ([score.artifacts * 400] Points)<BR><BR>
+
+	<U>THE BAD:</U><BR>
+	<B>Dead Crewmen:</B> [score.deadcrew] (-[score.deadcrew * 250] Points)<BR>
+	<B>Destroyed Silicons:</B> [score.deadsilicon] ([find_active_faction_by_type(/datum/faction/malf) ? score.deadsilicon * 500 : score.deadsilicon * -500] Points)<BR>
+	<B>AIs Destroyed:</B> [score.deadaipenalty] ([find_active_faction_by_type(/datum/faction/malf) ? score.deadaipenalty * 1000 : score.deadaipenalty * -1000] Points)<BR>
+	<B>Uncleaned Messes:</B> [score.mess] (-[score.mess] Points)<BR>
+	<B>Trash on Station:</B> [score.litter] (-[score.litter] Points)<BR>
+	<B>Station Power Issues:</B> [score.powerloss] (-[score.powerloss * 50] Points)<BR>
+	<B>Tiles destroyed by a singularity:</B> [score.turfssingulod] (-[round(score.turfssingulod/2)] Points)<BR>
+	<B>Bad diseases in living mobs:</B> [score.disease_bad] (-[score.disease_bad * 50] Points)<BR><BR>
+
+	<U>THE WEIRD</U><BR>"}
+/*	<B>Final Station Budget:</B> $[num2text(totalfunds,50)]<BR>"}
```

Comment:
Is there a reason this is commented our rather than deleted?

Reply:
I left the commented-out bits from the old code just in case for whatever reason someone wants to re-implement it easily. I didn't want to touch it. There's loads of bloat in there that I could remove, but I don't think commented-out code is harmful